File: online-judge-solutions/Codeforces/369A.cpp
//https://codeforces.com/problemset/problem/369/A
//Lang		:	GNU C++14
//Time		:	15 ms
//Memory	:	0 KB
#include<iostream>
using namespace std;
int main(){
	int n,m,k,a;
	while(cin>>n>>m>>k){
		int n1=0,n2=0;
		while(n--){cin>>a;a==1?n1++:n2++;}
		if(n1>=m){n1-=m;m=0;}
		else{m-=n1;n1=0;}
		n2=(n2<m+k)?0:n2-m-k;
		cout<<n1+n2<<endl;
	}
	return 0;
}

File: online-judge-solutions/Codeforces/369A.py
#https://codeforces.com/problemset/problem/369/A
#Lang		:	Python 3.8
#Time		:	46 ms
#Memory         :	0 KB
n,m,k=map(int,input().split())
n1=n2=0
a=list(map(int,input().split()))
for i in a:
    if(i==1): n1+=1
    else: n2+=1
if(n1>=m):
    n1-=m
    m=0
else:
    m-=n1
    n1=0
n2=0 if (n2<m+k) else n2-m-k
print(n1+n2)
